src/microbe_stage/MicrobeColony.cs

```diff
@@ -69,14 +68,23 @@ public float HexCount
     /// <summary>
     ///   The accumulation of all the colony member's <see cref="Microbe.UsedIngestionCapacity"/>.
     /// </summary>
+    /// <remarks>
+    ///   <para>
+    ///     This unfortunately is not cached as <see cref="Microbe.UsedIngestionCapacity"/> can change
+    ///     every frame.
+    ///   </para>
+    /// </remarks>
     [JsonIgnore]
     public float UsedIngestionCapacity
     {
         get
         {
-            if (hexCountDirty)
-                UpdateHexCount();
-            return usedIngestionCapacity;
+            var result = 0.0f;
+
+            foreach (var microbe in ColonyMembers)
+                result += microbe.UsedIngestionCapacity;
+
+            return result;
```

Comment:
```suggestion
            return ColonyMembers.Sum(c => c.UsedIngestionCapacity);
```
One liner that should do the the same thing